src/Client/BuzzHouse/Generator/SQLTable.cpp

```diff
@@ -118,7 +116,7 @@ void StatementGenerator::addTableRelation(
         [](const SQLColumn & c) { return !c.dmod.has_value() || c.dmod.value() != DModifier::DEF_EPHEMERAL; });
     for (const auto & entry : this->table_entries)
     {
-        std::vector<std::string> names;
+        std::vector<String> names;
```

Comment:
Not a big issue, but I've seen `std::vector<String>` quite a few times already. There's a `Strings` defined in `src/Core/Types.h`